### PR TITLE
Test update of bounding box in prime_db_schema.publications after WFS…

### PR DESCRIPTION
**Pull request by index-git**
…-T insert

Part of issue #159

- [x] Tests
- ~~Layman Test Client~~
- ~~Changlelog~~
- ~~Documentation~~

Optionally
- Add dependency to doc/dependencies.md

### CODE REVIEW

**Review by jirik — APPROVED** [review]

File: test/util.py [hunk 1]
```diff
@@ -73,3 +76,18 @@ def assert_same_bboxes(bbox1, bbox2, precision):
     assert len(bbox2) == 4, (bbox2, len(bbox2))
     for i in range(0, 3):
         assert abs(bbox2[i] - bbox1[i]) <= precision, (bbox1, bbox2, precision, i)
+
+
+def assert_wfs_bbox(workspace, layer, expected_bbox):
```
Comment (jirik) [inline test/util.py:81]: Create app context internally

File: test/util.py [hunk 2]
```diff
@@ -73,3 +76,18 @@ def assert_same_bboxes(bbox1, bbox2, precision):
...
+    wfs_layer = f"{workspace}:{layer}"
+    wfs_get_capabilities = wfs.get_wfs_proxy(workspace)
+    wfs_bbox_4326 = wfs_get_capabilities.contents[wfs_layer].boundingBoxWGS84
+    wfs_bbox_3857 = bbox_util.transform(wfs_bbox_4326, 4326, 3857, )
+    assert_same_bboxes(expected_bbox, wfs_bbox_3857, 0.00001)
+
+
+def assert_wms_bbox(workspace, layer, expected_bbox):
```
Comment (jirik) [inline test/util.py:89]: Create app context internally

File: src/layman/layer/geoserver/geoserver_test.py [hunk 3]
```diff
@@ -65,16 +49,16 @@ def test_geoserver_bbox():
     process_client.publish_workspace_layer(workspace, layer, style_file='sample/style/small_layer.qml')
 
     with app.app_context():
```
Comment (jirik) [inline src/layman/layer/geoserver/geoserver_test.py:51]: Remove with

File: src/layman/geoserver_proxy_test.py [hunk 4]
```diff
@@ -366,7 +366,16 @@ def do_test(wfs_query, attribute_names):
     client_util.delete_workspace_layer(username, layername1, headers=headers1)
 
 
-@pytest.mark.xfail(reason='Not yet implemented!')
+def assert_all_sources_bbox(workspace, layer, expected_bbox):
+    with app.app_context():
+        bbox = tuple(layman_util.get_publication_info(workspace, client_util.LAYER_TYPE, layer,
+                                                      context={'key': ['bounding_box']})['bounding_box'])
+        test_util.assert_same_bboxes(expected_bbox, bbox, 0)
+        test_util.assert_wfs_bbox(workspace, layer, expected_bbox)
```
Comment (jirik) [inline src/layman/geoserver_proxy_test.py:374]: Remove wfs and wms check from app context

**Review by jirik — APPROVED** [review]

**Review by jirik — APPROVED** [review]

**Comment by index-git** [comment]
Canceled, in other PRs.

**Pull request closed without merging**